test/governance/extensions/GovernorTimelockControl.test.js

```diff
@@ -109,67 +106,67 @@ contract('GovernorTimelockControl', function (accounts) {
         expect(await this.mock.timelock()).to.be.equal(this.timelock.address);
       });
 
-      it('nominal', async function () {
-        expect(await this.mock.proposalEta(this.proposal.id)).to.be.bignumber.equal('0');
-        expect(await this.mock.proposalNeedsQueuing(this.proposal.id)).to.be.equal(true);
-
-        await this.helper.propose();
-        await this.helper.waitForSnapshot();
-        await this.helper.vote({ support: Enums.VoteType.For }, { from: voter1 });
-        await this.helper.vote({ support: Enums.VoteType.For }, { from: voter2 });
-        await this.helper.vote({ support: Enums.VoteType.Against }, { from: voter3 });
-        await this.helper.vote({ support: Enums.VoteType.Abstain }, { from: voter4 });
-        await this.helper.waitForDeadline();
-        const txQueue = await this.helper.queue();
-        await this.helper.waitForEta();
-
-        const eta = web3.utils.toBN(await clockFromReceipt.timestamp(txQueue.receipt)).addn(delay);
-        expect(await this.mock.proposalEta(this.proposal.id)).to.be.bignumber.equal(eta);
-        expect(await this.mock.proposalNeedsQueuing(this.proposal.id)).to.be.equal(true);
-
-        const txExecute = await this.helper.execute();
-
-        expectEvent(txQueue, 'ProposalQueued', { proposalId: this.proposal.id });
-        await expectEvent.inTransaction(txQueue.tx, this.timelock, 'CallScheduled', { id: this.proposal.timelockid });
-        await expectEvent.inTransaction(txQueue.tx, this.timelock, 'CallSalt', {
-          id: this.proposal.timelockid,
-        });
-
-        expectEvent(txExecute, 'ProposalExecuted', { proposalId: this.proposal.id });
-        await expectEvent.inTransaction(txExecute.tx, this.timelock, 'CallExecuted', { id: this.proposal.timelockid });
-        await expectEvent.inTransaction(txExecute.tx, this.receiver, 'MockFunctionCalled');
-      });
+      // it('nominal', async function () {
```

Comment:
If those are meant to be removed, please, let's remove them, so it's less confusing